src/fr/tva.ts

```diff
@@ -80,8 +80,7 @@ const impl: Validator = {
     if (strings.isdigits(check)) {
       const sum = (12 + 3 * (parseInt(back, 10) % 97)) % 97;
 
-      const stringSum = String(sum).padStart(2, '0');
-      if (stringSum !== check) {
+      if (sum !== parseInt(check, 10)) {
```

Comment:
Would you mind adding a test case that handles this fix.

Reply:
Sure!